packages/filesystem/src/browser/file-resource.ts

```diff
@@ -109,14 +111,15 @@ export class FileResource implements Resource {
             return stat.value;
         } catch (e) {
             if (e instanceof TextFileOperationError && e.textFileOperationResult === TextFileOperationResult.FILE_IS_BINARY) {
-                if (await this.shouldOpenAsText('The file is either binary or uses an unsupported text encoding.')) {
+                if (await this.shouldOpenAsText(nls.localize('theia/filesystem/fileResource/binaryTitle', 'The file is either binary or uses an unsupported text encoding.'))) {
```

Comment:
```suggestion
                if (await this.shouldOpenAsText(nls.localizeByDefault('The file is not displayed in the editor because it is either binary or uses an unsupported text encoding.'))) {
```